### ISSUE TITLE
[Programmatic Access - Azure Databases Extension - Tree view]: "Learn more" link present under “Stats" tab is not descriptive enough.

### ISSUE THREAD

**Issue by ChandiniSmiley** [microsoft/vscode-cosmosdb#2663]
### **User Experience:** 

Users depending on screen readers will get affected as the screen reader does not pass the descriptive information for the link which may confuse the user about the different purpose of the links present on the page with same name. 

**Note:** Login Credentials must not be provided within the bug. 

### **Pre-requisite:** 

Azure database should be installed from extensions in VS code. 

### **Repro Steps** 

1. Open VS Code and navigate to azure databases button present on right pane. 
2. Navigate to anyone of the resource and expand it. Expand a database. Right click on any collection and select "Open Query editor(Preview)". 
3. Query editor screen appears. Run any sample query.  
4. Select stats tab. Navigate through all controls of stats tab. 
5. Verify if "Learn more" link present under "stats" tab is not descriptive enough. 

### **Actual Result:** 

Second "Learn more" link present under “Stats" tab is not descriptive enough i.e the text present before the link is “Index Utilization Information” but activating the link it opens “Index-metrics” page. 

Observation : Screen reader Narrator only announces “Learn more link”. 

### **Expected Result:** 

"Learn more" link present under "Stats" tab should be descriptive enough i.e when focus moves to it screen reader should announce as “Learn more about Index-metrics” link. 

### **MAS Reference:** 

[MAS 2.4.4 – Link Purpose (In Context)](https://aka.ms/MAS2.4.4)

### **Reference links:** 
 
- [Accessibility Insights!](https://accessibilityinsights.io/) - Identify accessibility bugs before check-in and make bug fixing faster and easier. 
- External Bug Process: If this bug belongs to external team, mark it as resolved/done and assign it back to the tester with notes on where to file/route the bug. For more information, please use this [External Bug Process (sharepoint.com)](https://microsoft.sharepoint.com/teams/cea11y/SitePages/External-Bug-Process.aspx?xsdata=MDN8MDF8fDE1MThmM2JlZGFjMDRlZDhiZmViYTUxMDA2M2NiZWI3fDcyZjk4OGJmODZmMTQxYWY5MWFiMmQ3Y2QwMTFkYjQ3fDF8MHw2Mzc3NDEyNTM2MTEzNzIwNzR8R29vZHxWR1ZoYlhOVFpXTjFjbWwwZVZObGNuWnBZMlY4ZXlKV0lqb2lNQzR3TGpBd01EQWlMQ0pRSWpvaUlpd2lRVTRpT2lJaUxDSlhWQ0k2TVRKOQ%3D%3D&sdata=MnlpSXlVMnpGT3cvZ0xJNnFWUWZWTGt6dmo1VkJMZFpIZHBvUUduNDhyMD0%3D&ovuser=72f988bf-86f1-41af-91ab-2d7cd011db47%2cv-kgeetala%40microsoft.com). “HCL Staff should not log any third-party external bugs. Should be routed to EDAD team”. 
- Please reach out to [C&AI Teams channel](https://apc01.safelinks.protection.outlook.com/?url=https://teams.microsoft.com/l/team/19%253a3cd071821b694f11b054d878453cd75f%2540thread.skype/conversations?groupId%3D30e3f968-15dd-4da2-bd8e-85f27e4af2c3%26tenantId%3D72f988bf-86f1-41af-91ab-2d7cd011db47&data=02%7c01%7cAklavyaM%40hcl.com%7c24e89c6ac26f42cfe40c08d7667e49f3%7c189de737c93a4f5a8b686f4ca9941912%7c0%7c0%7c637090564906844600&sdata=22JjBMoA1ID/KnYqFa/OB%2BrkTaLh5YYsGBd/7Bm7u5w%3D&reserved=0) for any process-related queries. 

### **Test Environment:**   

- OS Version: 24H2 (OS Build 26100.3476) 
- Visual Studio Code Version: 1.99.0 (user setup) 
- Tool: Accessibility Insights for Win32 
- Azure Databases for VS Code (Preview) Version : 0.25.2
- Matrix:Win32+Narrator 

https://github.com/user-attachments/assets/817bce64-49a8-4fc4-95ed-e43cada14216

**Comment by ChandiniSmiley** [comment]
@mkrueger , As we checked in the below version, the issue is still repro. Hence reactivating the bug.

**Test Environment:**

- OS Version: 24H2 (OS Build 26100.3476)
- Visual Studio Code Version: 1.99.0 (user setup)
- Tool: Accessibility Insights for Win32
- Azure Databases for VS Code (Preview) Version : 0.25.2
- Matrix:Win32+Narrator

**Comment by sevoku** [comment]
@ChandiniSmiley please use the latest version built from main. You can find it by navigating here: https://github.com/microsoft/vscode-cosmosdb/actions/workflows/main.yml?query=branch%3Amain, selecting the latest (first) build and download the Artifacts.zip from the Summary tab. The latest version is 0.27.0.alpha.

**Comment by ChandiniSmiley** [comment]
@sevoku , Verified the issue on the latest build, the issue is still repro. Please refer the attached video.

https://github.com/user-attachments/assets/b30ce64f-cade-402a-938a-0cc781d9b03c

**Comment by sevoku** [comment]
This is now fixed in main though #2710

**Comment by ChandiniSmiley** [comment]
As verified in the below version, the issue is fixed. Hence, closing the bug.

https://github.com/microsoft/vscode-cosmosdb/actions/runs/15868041052/job/44738540612
